gui/cache.go

```diff
@@ -290,10 +289,10 @@ func (c *Cache) updatePayments(pendingPmts []*pool.Payment, archivedPmts []*pool
 		accountID := p.Account
 		pendingPayments[accountID] = append(pendingPayments[accountID],
 			&pendingPayment{
-				WorkHeight:    fmt.Sprint(p.Height),
-				WorkHeightURL: blockURL(c.blockExplorerURL, p.Height),
-				Amount:        amount(p.Amount),
-				CreatedOn:     formatUnixTime(p.CreatedOn),
+				WorkHeight:             fmt.Sprint(p.Height),
+				WorkHeightURL:          blockURL(c.blockExplorerURL, p.Height),
+				Amount:                 amount(p.Amount),
+				EstimatedPaymentHeight: fmt.Sprint(p.EstimatedMaturity + 1),
```

Comment:
The string conversions here will not be needed if the types are their default types.